mdal/frmts/mdal_ascii_dat.cpp

```diff
@@ -226,9 +227,30 @@ void MDAL::DriverAsciiDat::loadNewFormat( std::ifstream &in,
       if ( quoteIdx1 != std::string::npos && quoteIdx2 != std::string::npos )
         group->setName( line.substr( quoteIdx1 + 1, quoteIdx2 - quoteIdx1 - 1 ) );
     }
+    else if ( cardType == "RT_JULIAN" && items.size() >= 2 )
+    {
+      size_t quoteIdx1 = line.find( " " );
+      size_t quoteIdx2 = line.size();
+      if ( quoteIdx1 != std::string::npos && quoteIdx2 != std::string::npos )
+        referenceTime = ( line.substr( quoteIdx1 + 1, quoteIdx2 - quoteIdx1 - 1 ) );
+    }
+    else if ( cardType == "TIMEUNITS" && items.size() >= 2 )
+    {
+      if ( !group )
+      {
+        debug( "TIMEUNITS card for no active dataset!" );
+        EXIT_WITH_ERROR( MDAL_Status::Err_UnknownFormat );
+      }
+
+      size_t quoteIdx1 = line.find( " " );
+      size_t quoteIdx2 = line.size();
+      if ( quoteIdx1 != std::string::npos && quoteIdx2 != std::string::npos )
+        group->setMetadata( "TIMEUNITS", line.substr( quoteIdx1 + 1, quoteIdx2 - quoteIdx1 - 1 ) );
+    }
     else if ( cardType == "TS" && items.size() >= 3 )
     {
       double t = toDouble( items[2] );
+      t = convertTimeDataToHours( t, group->getMetadata( "TIMEUNITS" ) );
```

Comment:
give it a default value of seconds, so in case there is no TIMEUNITS in the file it still gets the same result as before